content/en/observability_pipelines/_index.md

```diff
@@ -43,31 +43,80 @@ Datadog recommends you update Observability Pipelines Worker (OPW) with every mi
+
+{{< img src="observability_pipelines/templates.png" alt="The Observability Pipelines UI showing the six templates" style="width:100%;" >}}
+
+The [templates](#out-of-the-box-templates) are purpose-built for the following use cases:
+
+#### Log Volume Control
+
+Raw logs are noisy and only some logs are useful for further search and analysis during investigations. Use the Log Volume Control template to determine which logs to send to your indexed solution, such as a SIEM or log management solution. This helps you to increase the value of your indexed logs and also remain within your planned budget.
+
+#### Dual Ship Logs
+
+As your organization grows, your observability needs for different use cases, such as security, archiving, and log management also change. This could mean having to trial different archiving, SIEM, and log management solutions. However, managing log pipelines to different solutions can get complicated. Use the Dual Ship Logs template to centrally aggregate, process, and send copies of your logs to different destinations.
+
+#### Archive Logs
+
+Use the Archive Logs template to store logs in a cloud storage solution (Amazon S3, Google Cloud Storage, or Azure Storage). The archived logs are stored in a Datadog-rehydratable format, so that they can be rehydrated in Datadog as needed. This is useful when:
+
+- You have a high volume of noisy logs, but might need to index them in Datadog Log Management ad hoc for an investigation.
+- You are migrating to Datadog Log Management and want to have historical logs after completing the migration.
+- You have a retention policy to fulfill compliance requirements but don't necessarily need to index those logs.
+
+#### Split Logs
+
+When you have logs from different services and applications, you might need to send them to different downstream services for querying, analysis, and alerting. For example, you might want to send security logs to a SIEM solution and DevOps logs to Datadog. Use the Split Logs template to pre-process your logs separately for each destination before sending them downstream.
+
+#### Sensitive Data Redaction
+
+Use the Sensitive Data Redaction template to detect and redact sensitive information on premises. The Observability Pipelines sensitive data scanner processor provides 70 out-of-the-box scanning rules, but you can also create your own custom scanning rules using regex. The OOTB rules recognize standard patterns such as credit card numbers, email addresses, IP addresses, API and SSH keys, and access tokens.
+
+#### Log Enrichment
+
+Your organization's different services, systems, and application all generate logs containing layers of information and in different formats. This can make it difficult to extract the data you need when searching and analyzing the data for an investigation. Use the Log Enrichment template to standardize your logs and enrich it with information, from example, from a reference table.
```

Comment:
```suggestion
Your organization's different services, systems, and applications all generate logs containing layers of information and in different formats. This can make it difficult to extract the data you need when searching and analyzing the data for an investigation. Use the Log Enrichment template to standardize your logs and enrich them with information, such as data from a reference table.
```